qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -880,6 +890,20 @@ wait_for_status_in_pool_dir()
     return 1
 }
 
+wait_for_replay_status_in_pool_dir()
```

Comment:
As I wrote later, I would like `wait_for_replaying_status_in_pool_dir` more, but I am fine with the current name too if you like it more.

Reply:
done